iRecruit-express-app/database.js

```diff
@@ -1,6 +1,39 @@
 import { Sequelize } from 'sequelize';
+import memcache from './memcache';
+import { User } from './models';
 
 export const sequelize = new Sequelize('capstone', 'henrywis', '@Henrywis1', {
   host: 'localhost',
   dialect: 'postgres'
-});
\ No newline at end of file
+});
+
+
+// Constant for 5 minutes in milliseconds
+const FIVE_MINUTES = 5 * 60 * 1000;
+
+export async function fetchUserDataFromDatabase(userId) {
+  // Check if data is in cache
+  const cachedData = memcache.get(`user:${userId}`);
```

Comment:
This is an example of a proper usage of caching, we are returning a specific database payload specific to a user.  

We however also need to consider what happens when this user's data changes, we need to either invalidate this cache because its outdated or force a refresh somehow.

Reply:
Got it!